lib/order.rb

```diff
@@ -19,4 +20,28 @@ def add_selection_to_order
     @shopping_cart << @selection
   end
 
+  def work_out_price
+    @shopping_cart.each do |food_item|
+      food_item.each do |item, quantity|
```

Comment:
could refactor for better readability.

Reply:
Yes, I'll elaborate further on this: `each`es inside `each`es gets confusing. There is a cleaner way to look up whether an item exists on the menu, and look up an items price: splitting those functionalities out into methods. But where should those methods live? Ideally, on your Menu class: that preserves encapsulation, i.e. only the methods that have a direct need-to-know about the menu list are able to know about it.